back-end/e-learning/database/migrations/2022_02_21_061923_create_words_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateWordsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('words', function (Blueprint $table) {
+            $table->id();
+            $table->integer('category_id')->unsigned();
```

Comment:
Please add that `category_id` is a foreign key. and add `onDelete cascade`
https://laravel.com/docs/8.x/migrations#foreign-key-constraints